source/tasks/CreateOctopusRelease/CreateOctopusReleaseV6/input-parameters.test.ts

```diff
@@ -0,0 +1,97 @@
+import { Logger } from "@octopusdeploy/api-client";
+import { TaskWrapper } from "tasks/Utils/taskInput";
+import { getInputParameters } from "./input-parameters";
+import * as tasks from "azure-pipelines-task-lib/task";
+
+export class MockTaskWrapper implements TaskWrapper {
```

Comment:
This can be removed and an import statement added `import { MockTaskWrapper } from "../../Utils/MockTaskWrapper";`

Reply:
👍